scripts/http-vuln-cve2014-3704.nse

```diff
@@ -216,9 +216,14 @@ local function do_sql_query(host, port, uri, user)
     }
   }
   local res = http.post(host, port, uri .. "/user/login", opt, nil, r)
-  --TODO: Check return status
 
-  return user, passwd
+  if string.match( res.body, 'mb_strlen() expects parameter 1 to be string' ) then
```

Comment:
I checked this condition against both the vulnerable and patched instances and found it to be correct. Although Metasploit module works great, this is a more accurate condition.